<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SimSun-ExtB</family>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>RoboGame 2020</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(231, 233, 238)</string>
   </property>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>RoboGame 2020 “共抗疫情”比赛</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>890</y>
      <width>1541</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Boxs">
     <item>
      <widget class="QLabel" name="Box1">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RG.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Box2">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RG.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Box3">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RG.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Box4">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RY.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Box5">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RY.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Box6">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RR.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>1201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Times">
     <item>
      <widget class="QLabel" name="LTime">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>70</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(40, 64, 121);</string>
       </property>
       <property name="text">
        <string>05:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="CTime">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>70</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>01:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RTime">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>70</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(121, 31, 34);</string>
       </property>
       <property name="text">
        <string>05:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>RTime</zorder>
    <zorder>CTime</zorder>
    <zorder>LTime</zorder>
   </widget>
   <widget class="QLabel" name="LComment">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>480</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:blue

</string>
    </property>
    <property name="text">
     <string>蓝队准备中...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="RComment">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>480</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red</string>
    </property>
    <property name="text">
     <string>红队准备中...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="TimeUp">
    <property name="geometry">
     <rect>
      <x>-1920</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>640</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,0,0,150);
background-color:rgb(255,255,255,200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LScorePlus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>329</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>96</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(230, 231, 235);
background-color:rgb(17, 117, 192);
border:none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RScorePlus">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>330</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>96</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(230, 231, 235);
background-color:rgb(255, 111, 111);
border:none;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LScoreMinus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>96</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="RScoreMinus">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>180</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>96</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>681</y>
      <width>601</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Blocks">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Block1">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:solid;
border-width:0.5px;
border-color: hsl(0,0,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/figs/RY.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Block2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:solid;
border-width:0.5px;
border-color: hsl(0,0,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Block3">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:solid;
border-width:0.5px;
border-color: hsl(0,0,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>1201</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Buttons">
     <item row="0" column="0">
      <widget class="QPushButton" name="LStop">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(40, 64, 121);
background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(17, 117, 192, 255), stop:1 rgba(255, 255, 255, 255));
border:none;</string>
       </property>
       <property name="text">
        <string>蓝队停止计时</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Start">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(97, 106, 88);
background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(149, 164, 141, 255), stop:1 rgba(255, 255, 255, 255));
border:none;
</string>
       </property>
       <property name="text">
        <string>开始计时</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="RStop">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(121, 31, 34);
background-color:qlineargradient(spread:reflect, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 111, 111, 255), stop:1 rgba(255, 255, 255, 255));
border:none;</string>
       </property>
       <property name="text">
        <string>红队停止计时</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="Random">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(97, 106, 88);
background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(149, 164, 141, 255), stop:1 rgba(255, 255, 255, 255));
border:none;</string>
       </property>
       <property name="text">
        <string>生成随机序列</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="Reset">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(97, 106, 88);
background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(149, 164, 141, 255), stop:1 rgba(255, 255, 255, 255));
border:none;</string>
       </property>
       <property name="text">
        <string>复位</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="Countdown">
       <property name="maximumSize">
        <size>
         <width>360</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>OpenHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(97, 106, 88);
background-color:qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(149, 164, 141, 255), stop:1 rgba(255, 255, 255, 255));
border:none;</string>
       </property>
       <property name="text">
        <string>1分钟倒计时</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>1720</x>
      <y>0</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/figs/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>RComment</zorder>
   <zorder>LComment</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>Title</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>LScorePlus</zorder>
   <zorder>RScorePlus</zorder>
   <zorder>LScoreMinus</zorder>
   <zorder>RScoreMinus</zorder>
   <zorder>Logo</zorder>
   <zorder>TimeUp</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
